scripts/import_assets/import_assets.py

```diff
@@ -29,11 +30,22 @@ def read_and_create_assets(args):
             dict_fields = {
                 "name": name,
                 "type": asset_type,
-                "business_impact": business_impact,
-                "details": details,
-                "tags": tags,
-                "hosts_personal_data": str_to_bool(hosts_personal_data)
+                "tags": tags
             }
+
+            if details != '':
+                dict_fields.update({'details': details})
+
+            if business_impact != '':
+                dict_fields.update({'business_impact': business_impact})
+            else:
+                dict_fields.update({'business_impact': 'neutral'})
+
+            if hosts_personal_data != '':
+                dict_fields.update({'hosts_personal_data': str_to_bool(hosts_personal_data)})
```

Comment:
Instead of using the `.update()` method, it would be more clear to use just brackets:

```
dict_fields['hosts_personal_data'] = str_to_bool(hosts_personal_data)
``` 
The .update() will be more useful if you have multiple fields to update, not just one. :+1:

Reply:
Done!